Dockerfile

```diff
@@ -1,9 +1,9 @@
 # Build step for check-payload tool
-FROM registry.access.redhat.com/ubi9/go-toolset:9.5 as check-payload-build
+FROM quay.io/projectquay/golang:1.23 as check-payload-build
```

Comment:
I'm worried about using this image since it's not from a repository that's trusted by Konflux (if I'm not mistaken), and we're working towards onboarding this image onto Konflux.

Reply:
Just updated to use `brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23`